petstore/frontend/react/src/comments/Comments.js

```diff
@@ -0,0 +1,183 @@
+import React, {Component} from 'react';
+import {string} from 'prop-types';
+import AddComment from './AddComment';
+import Thread from './Thread';
+import config from '../config';
+
+class Comments extends Component {
+  state = {
+    enabled: false,
+    reply: {
+      content: '',
+      poster: ''
+    },
+    threads: []
+  };
+
+  async componentDidMount() {
```

Comment:
You'll see lots of use of async/await in the code now. I find that much easier to read than Promise chains.